xcell/cls/data.py

```diff
@@ -249,6 +249,12 @@ def check_toeplitz(self, dtype):
 
     def get_mapper(self, tr):
         config = self.data['tracers'][tr]
+        nside = self.data['sphere']['nside']
+        if 'nside' not in config:
+            config['nside'] = nside
```

Comment:
I.e. I understand now what you're doing here is adding nside to the mapper's config if it's not there. But this only works if you're gonna get cls. If you just want to instantiate a specific mapper, you need to create a config dictionary that includes `nside`, right?

What I propose is that we change the the constructor of all mappers so that instead of taking only a `config` dictionary, they take `__init__(config, nside)`.

Reply:
But maybe I'm being pedantic, we can just drop it...

Reply:
But if your going to instantiate a specific mapper, you will have to create the config dictionary anyway, won't you? What's the advantage you see on making it a separate argument of the constructor?